check/setup.go

```diff
@@ -314,10 +336,12 @@ func (check *HealthCheck) closeConnection(deleteTopicIfPresent bool) error {
 			return fmt.Errorf("error while taking cluster lock: %w", err)
 		}
 		log.Info("lock acquired to close the connection")
-		if err = check.deleteTopic(zkConn, chroot, check.config.topicName, check.partitionID); err != nil {
+
+    if err = check.deleteTopic(check.config.topicName); err != nil {
 			log.Errorf("error while deleting topic %s: %s", check.config.topicName, err)
 		}
-		if err = check.deleteTopic(zkConn, chroot, check.config.replicationTopicName, check.replicationPartitionID); err != nil {
+		if err = check.deleteTopic(check.config.replicationTopicName); err != nil {
```

Comment:
not related to this PR, but I thought that the replication topic would not be deleted when closing the connection...